x/staging/subspaces/client/cli/tx.go

```diff
@@ -0,0 +1,249 @@
+e.g 1) %s tx subspaces create 4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e "mooncake"
+	2) %s tx subspaces create 4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e "mooncake" true
+`, version.AppName, version.AppName),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			subspaceID := args[0]
+			subspaceName := args[1]
+			open := false
+			if len(args) > 2 {
+				open, err = strconv.ParseBool(args[2])
+				if err != nil {
+					return fmt.Errorf("open field can only be true or false")
+				}
+			}
+
+			msg := types.NewMsgCreateSubspace(subspaceID, subspaceName, clientCtx.FromAddress.String(), open)
+			if err = msg.ValidateBasic(); err != nil {
+				return fmt.Errorf("message validation failed: %w", err)
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+	return cmd
+}
+
+func GetCmdAddSubspaceAdmin() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add-admin [address] [subspace-id]",
+		Args:  cobra.ExactArgs(2),
+		Short: "Add a new admin to the subspace with the given [subspace-id]",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			newAdminAddress, err := sdk.AccAddressFromBech32(args[0])
+			if err != nil {
+				return err
+			}
+
+			subspaceID := args[1]
+
+			msg := types.NewMsgAddAdmin(subspaceID, newAdminAddress.String(), clientCtx.FromAddress.String())
+			if err = msg.ValidateBasic(); err != nil {
+				return fmt.Errorf("message validation failed: %w", err)
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+
+	return cmd
+}
+
+func GetCmdRemoveSubspaceAdmin() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "remove-admin [address] [subspace-id]",
+		Args:  cobra.ExactArgs(2),
+		Short: "Remove an existent admin from the subspace with the given id",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			existentAdminAddress, err := sdk.AccAddressFromBech32(args[0])
+			if err != nil {
+				return err
+			}
+
+			subspaceID := args[1]
+
+			msg := types.NewMsgRemoveAdmin(subspaceID, existentAdminAddress.String(), clientCtx.FromAddress.String())
+			if err = msg.ValidateBasic(); err != nil {
+				return fmt.Errorf("message validation failed: %w", err)
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+
+	return cmd
+}
+
+func GetCmdRegisterUser() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "register-user [address] [subspace-id]",
+		Args:  cobra.ExactArgs(2),
+		Short: "Register a user inside the subspace with the given [subspace-id] to let him post in it",
```

Comment:
No need to tell what registration does, since it might change in the future and we don't want to have to update this doc 
```suggestion
		Short: "Register a user inside the subspace with the given id",
```